data/css/all-aboard.css

```diff
@@ -82,7 +86,7 @@ a:focus {
 }
 
 .sidebar-content.values footer {
-    padding-top: 40px;
+    padding-top: 0;
```

Comment:
As we are turning off top padding for all the values and utility sidebars, we can simplify this to one rule so, remove:
```
.sidebar-content.values footer
```
and
```
.sidebar-content.utility footer
```
and replace with just
```
.sidebar-content footer {
    padding-top: 0;
}
```